private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -791,6 +793,92 @@ export class TemplateServiceClient {
     return { success: true, result: resultTemplates };
   }
 
+  /**
+   * @public
+   * Get entry point.
+   * Returns the specified template with the data bound into the template JSON.
+   * If a version is passed in, it will update the template JSON of that version.
+   * If not version is passed, it will update the latest version.
+   * @param {string} templateId - unique template id
+   * @param {JSON} data - data json to bound with template json
+   * @param {string} version - version number, used with templateId
+   * @param {boolean} isClient - used to ignore updating the hit number on a template
+   * @returns Promise as valid json
+   */
+  public async bindData(
+    token: string,
```

Comment:
for public functions token should be optional so that we can use the default one stored in the authprovider class

Reply:
kk!